### ISSUE TITLE
localhost: CORS errors when accessing API

### ISSUE THREAD

**Issue by elbotho** [serlo/frontend#1997]
on several places cors errors occure (loading comments, [loading repository history](http://localhost:3000/entity/repository/history/1555), …) when accessing from localhost.

@hugotiburtino do we need to proxy non authed request now as well?

error message:
> Access to XMLHttpRequest at 'https://api.serlo-staging.dev/graphql' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: The 'Access-Control-Allow-Origin' header has a value 'https://serlo-staging.dev' that is not equal to the supplied origin.

**Comment by hugotiburtino** [comment]
> do we need to proxy non authed request now as well?

I don't think so, cors handling in CF worker was changed in the meanwhile, we'll take a look at that and fix it (ping @kulla)
